docs/conf/example-proposal.rst

```diff
@@ -0,0 +1,50 @@
+Create a catchy title that promises pratical lessons
+====================================================
+
+Abstract
+--------
+
+Incldue a brief story, typically two or three paragraphs, that describes your personal work experience with the topic. Make sure the abstract appeals to our audience of documentarians. We're suspicious of proposals that look like they belong at other conferences.
+
+Include a list of things that our audience can learn from your talk, such as:
+
+- Lessons that the audience can apply in their own work
+- Things that audiences should research further
+- Spoilers that provide hints about the talk
+
+Avoid walls of text.
+
+Who and Why
+-----------
+
+Use this opportunity to empathize with your audience. Help them think: "Oh yes, this talk could help me when I do X in my work!"
+
+Our audience creates documentation primarily for software. Given the variety of tools used for software documentation, we rarely accept talks that include tooling.
+
+Our audence goes beyond the technical writing community. Here's a typical demographic distribution of people who attend our confernces:
+
+* Technical Writers (60%)
+* Developers (10%)
+* Support Staff (10%)
+* Managers (10%)
+* Community Contributors, Enthusiasts & Other Folks (10%)
+
+Other Information
+-----------------
+
+Based on your background, use this section to describe your qualifications to the selection committee.
+
+We welcome talks from first-time speakers, from industry experts, and from everyone in between. Whatever your background and experience, we prefer hearing about new approaches than about tried-and-tested technology. We especially welcome talks from underrepresented groups within the tech community.
```

Comment:
s/than/rather than (or some edit or other to larger sentence that I couldn't quite get to ...)

Reply:
I took that from some other text on the site (which should also be modified).

Reply:
I've also included `rather` in the prague/2020/cfp.rst wording, from where I took this. While it should be included in other past cfp.rst files, that's in the past. (Do we have an actual cfp.rst template, or is that a different issue?)

Reply:
We leave past years as they are, and just copy/update the current year.